SETUP/mk_loc_lang_names.sh

```diff
@@ -1,4 +1,5 @@
 #!/bin/bash
```

Comment:
We should just delete this. According to git log it's been in our codebase for 14 years and never used.

Reply:
`rm`-inating!